src/vte.c

```diff
@@ -272,11 +272,9 @@ void vte_init(void)
 #ifdef __APPLE__
 			"libvte-2.91.0.dylib", "libvte-2.91.dylib",
 			"libvte2_90.9.dylib", "libvte2_90.dylib",
-#else
+#endif
 			"libvte-2.91.so", "libvte-2.91.so.0",
 			"libvte2_90.so", "libvte2_90.so.9",
-			"libvte.so", "libvte.so.9", "libvte.so.8", "libvte.so.4",
-#endif
```

Comment:
Here I dropped the GTK2-only VTEs, and restored the fact that before your changes, on macos the .so were included in the list.  I have no idea if it makes sense, but I'd rather be conservative here.